core/src/main/scala/com/microsoft/azure/synapse/ml/logging/SynapseMLLogging.scala

```diff
@@ -40,6 +42,11 @@ object SynapseMLLogging extends Logging {
     logInfo(s"metrics/ ${mapToPrint.toJson.compactPrint}")
   }
 
+  def logMessage(message: String): Unit = {
+    val scrubbedMessage = SASScrubber.scrub(message)
+    logInfo(s"sending $scrubbedMessage")
```

Comment:
```suggestion
    logInfo(SASScrubber.scrub(message))
```

Reply:
no need to have sending in this more general API

Reply:
Great. Thanks for the suggestion. That append of a prefix "sending" slipped in unintentionally.